### ISSUE TITLE
Uninitialized HybridArray

### ISSUE THREAD

**Issue by prittjam** [mateuszbaran/HybridArrays.jl#57]
Is it possible to directly construct uninitialized memory like this, 

```
 M = HybridArray{Tuple{StaticArrays.Dynamic(),9}}(undef,4,9))
```

**Comment by mateuszbaran** [comment]
Currently that's not possible but it looks like a good idea.

**Comment by mateuszbaran** [comment]
Fixed in 0.4.14, I forgot to tag it earlier.
